tests/test_au/tests.py

```diff
@@ -195,6 +196,56 @@ def test_raises_error_for_invalid_abn(self):
         self.assertRaises(ValidationError, lambda: validator(invalid_abn))
 
 
+class AULocalFlavorAUCompanyNumberFieldValidatorTests(TestCase):
+
+    def test_no_error_for_a_valid_acn(self):
+        """Test a valid ACN does not cause an error."""
+        valid_acn = '604327504'
+
+        validator = AUCompanyNumberFieldValidator()
+
+        validator(valid_acn)
+
+    def test_raises_error_for_acn_containing_a_letter(self):
+        """Test an ACN containing a letter is invalid."""
+        invalid_acn = '60432750A'
+
+        validator = AUCompanyNumberFieldValidator()
+
+        self.assertRaises(ValidationError, lambda: validator(invalid_acn))
+
+    def test_raises_error_for_too_short_acn(self):
+        """Test an ACN with fewer than nine digits is invalid."""
+        invalid_acn = '60432750'
+
+        validator = AUCompanyNumberFieldValidator()
+
+        self.assertRaises(ValidationError, lambda: validator(invalid_acn))
+
+    def test_raises_error_for_too_long_acn(self):
+        """Test an ACN with more than nine digits is invalid."""
+        invalid_acn = '6043275040'
+        validator = AUCompanyNumberFieldValidator()
+        self.assertRaises(ValidationError, lambda: validator(invalid_acn))
+
+    def test_raises_error_for_whitespace(self):
+        """Test an ACN can be valid when it contains whitespace."""
+        # NB: Form field should strip the whitespace before regex valdation is run.
```

Comment:
valdation -> validation

I see that's also in the copy & pasted code so maybe you can also fix the typo there as well.

As this comment points out, the whitespace is stripped in the form. It would be nice to have a test for the form as well - at least testing the expected functionality that isn't tested with the validation tests.